src/pages/proposals/ProposalDetail.vue

```diff
@@ -836,8 +836,8 @@ export default {
     .col-12.col-md-3(:class="{ 'q-pl-md': $q.screen.gt.sm }")
       widget.bg-primary(v-if="status === 'drafted'")
         h2.h-h4.text-white.leading-normal.q-ma-none Your proposal is on staging
-        p.h-b2.q-mt-xl.text-disabled Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-        q-btn.q-mt-xl.text-primary.text-bold.full-width( @click="onPublish(proposal)" color="white" text-color='primary' no-caps rounded) Publish
+        p.h-b2.q-mt-xl.text-disabled That means your proposal is not publish to the blockchain yet. You can still make changes to it, when you feel ready click "Publish" and the voting period will start.
+        q-btn.q-mt-xl.text-primary.text-bold.full-width( @click="onPublish(proposal)" color="white" text-color='primary' no-caps rounded) Publish to Staging
```

Comment:
I think this text should be on the proposal creation the last step and not here because you can't publish to staging if it is already on staging 
![image](https://user-images.githubusercontent.com/8117939/169359095-e21a540c-8abc-4eae-8dea-6d2dee7119f7.png)

@arsenijesavic